roles/maas_corosync/template/corosync.conf.j2

```diff
@@ -0,0 +1,41 @@
+system {
+    allow_knet_handle_fallback: no
+}
+
+totem {
+    version: 2
+    secauth: off
+    cluster_name: postgres_pacemaker
+    transport: udpu
+}
+
+logging {
+    fileline: off
+    to_stderr: yes
+    to_logfile: yes
+    logfile: /var/log/corosync/corosync.log
+    to_syslog: yes
+    debug: off
+    logger_subsys {
+        subsys: QUORUM
+        debug: off
+    }
+}
+
+
+nodelist {
+{% for i, pacemaker_host in enumerate(groups['pacemaker']) %}
```

Comment:
I'm not sure if enumerate exists as a filter, but ignore if I'm wrong about that